main.go

```diff
@@ -201,8 +237,19 @@ func getRepo(directory string, credentials []IssueAPI) (string, IssueAPI, error)
 			"associated with it.")
 	}
 
+	aliases, err := getURLAliases()
+	if err != nil {
+		return "", nil, err
+	}
+
 	urlString := url.String()
 
+	for key, value := range aliases {
+		if strings.Contains(urlString, key) {
+			urlString = strings.Replace(urlString, key, value, 1)
```

Comment:
I think we should break the loop after the first match. This code can potentially chain several replacements which may result in a pretty weird behavior.